src/components/discovery/DataTypeExplorationModal.js

```diff
@@ -76,7 +76,9 @@ class DataTypeExplorationModal extends Component {
                     <Radio.Button value="table"><Icon type="table" /> Table Detail View</Radio.Button>
                 </Radio.Group>
                 <Tabs>
-                    {Object.values(this.props.dataTypes).flatMap(ds => (ds.items ?? []).map(dataType =>
+                    {Object.values(this.props.dataTypes).flatMap(ds => (ds.items ?? [])
+                        .filter(dataType => dataType.queryable && dataType.count > 0)
```

Comment:
this is good for now, but in the future we will need to filter by data type at the level of the dataset, not the whole of the instance. maybe that can be a separate ticket?

Reply:
also, if queryable isn't set, it should default to true

so this would look like

```js
.filter(dataType => (dataType.queryable ?? true) && dataType.count > 0)
```

Reply:
Let's handle this in a separate ticket. As for the default value of queryable, you're absolutely right. I've updated the code to treat queryable as true when it's not set. Thanks for your insights!!